website/docs/docs/on-premises/releases/1.34.0.md

```diff
@@ -0,0 +1,22 @@
+---
+title: 1.34.0
+---
+
+## Release date
+
+22 October 2024
+
+## Features
+
+N/A
+
+## Fixes
+
+* fixes an issue where deleted idp/saml users who somehow still has useridentities records in the db, facing issues logging in.
+* updated packages to fix CVEs ( webrick, puma, openssl-dev, libssl3 openssl libcrypto3 )	
```

Comment:
```suggestion
* updated packages to fix multiple CVEs related to webrick, puma, openssl-dev, libssl3, openssl, libcrypto3. 
```

Reply:
Ideally, we list the CVEs we've fixed (our customers will be looking for these).